assets/src/components/amp-support/index.js

```diff
@@ -0,0 +1,453 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { Component } from '@wordpress/element';
+import { Button, ExternalLink } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import { __, sprintf } from '@wordpress/i18n';
+import { ListItem } from '../list-item';
+import { Selectable } from '../selectable';
+import { AMPNotice } from '../amp-notice';
+import ClipboardButton from '../clipboard-button';
+
+export class AMPSupport extends Component {
```

Comment:
Why are we using class components and not function components here (like in other parts of the codebase)?

Reply:
We can use the function component. I'll update it.